barretenberg/acir_tests/sol-test/src/index.js

```diff
@@ -47,6 +49,9 @@ const [test, verifier] = await Promise.all([
   fsPromises.readFile(verifierPath, encoding),
 ]);
 
+// 1. figure out how to deploy the libraries
```

Comment:
stale, although a note that the two libraries are now deployed separately is probably a good move

Reply:
ah i see this note below

Reply:
yeah there is a note there, will remove thx